.github/actions/pytest_run_cuda/action.yml

```diff
@@ -10,7 +10,7 @@ runs:
   steps:
     - name: Ccuda tests with pytest
       run: |
-        python -m pytest -n auto -rx -m "not (parallel or xdist_incompatible) and ccuda" --ignore=symbolic --ignore=ndarrays
+        python -m pytest -n auto -rx -m "not (parallel or xdist_incompatible) and ccuda" --ignore=tests/symbolic --ignore=tests/ndarrays --ignore=tests/cuda_ndarrays
         pyccel-clean
       shell: ${{ inputs.shell_cmd }}
-      working-directory: ./tests
+      working-directory: ./
```

Comment:
Please revert the dangerous changes in this file (which I assume come from a bad merge). See Issue #1344 and PR #1348 for more details